bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/TextLayout.java

```diff
@@ -1992,16 +1994,16 @@ public int getLevel (int offset) {
  */
 public Rectangle getLineBounds (int lineIndex) {
 	checkLayout();
-	return DPIUtil.autoScaleDown(getDevice(), getLineBoundsInPixels(lineIndex));
+	return getLineBoundsInPixels(lineIndex);
```

Comment:
Does getLineBoundsInPixels return pixel values? If yes, scaling down would be correct, if not, we should change the name

Reply:
Nope not anymore, all the scaling has been removed now.